public/main.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+let subReddits = [`https://www.reddit.com/r/Unity3D.json`, `https://www.reddit.com/r/blender.json`,
```

Comment:
Typically if you have a lot of items in an array, it's best to put each entry on their own line: 
```
let subReddits = [
 one,
 two, 
 three,
 etc...
];
```